src/Data/Fin/Permutation.agda

```diff
@@ -152,6 +156,31 @@ remove {m} {n} i π = permutation to from
     punchOut {i = πʳ i} {punchIn (πʳ i) j}                         _  ≡⟨ punchOut-punchIn (πʳ i) ⟩
     j                                                                 ∎
 
+-- lift: takes a permutation m → n and creates a permutation (suc m) → (suc n)
+-- by mapping 0 to 0 and applying the input permutation to everything else
+liftₚ : ∀ {m n} → Permutation m n → Permutation (suc m) (suc n)
+liftₚ {m} {n} π = permutation to from
+  record
+  { left-inverse-of = left-inverse-of
+  ; right-inverse-of = right-inverse-of
+  }
+  where
+    to : Fin (suc m) → Fin (suc n)
```

Comment:
No need to indent again after the `where`. Same comment in other functions in this file